config/runtimes/kserve-huggingfaceserver-multinode.yaml

```diff
@@ -21,17 +21,22 @@ spec:
       - "bash"
       - "-c"
       - |
-        export MODEL=${MODEL_ID}
+        export MODEL_DIR_ARG=""
         if [[ ! -z ${MODEL_DIR} ]]
```

Comment:
It checks the same value(`MODEL_DIR`), is it right? should be `MODEL_ID`?

Reply:
I fixed.